src/routers/handlers/upload/upload.ts

```diff
@@ -15,6 +15,7 @@ export class UploadHandler extends GenericHandler {
     constructor(private accountsFilingService: AccountsFilingService, private transactionService: TransactionService) {
         super({
             title: '',
+            viewName:"upload",
```

Comment:
There needs to be a space between the colon and the quotation mark in order to comply with the linting rules.

You can run the linter with `npm run lint` and you can use `npm run lint:fix` to fix and automatically correctable issues such as this one.

If the linter is not happy it will fail in the pipeline.

```suggestion
            viewName: "upload",
```